.github/workflows/ci.yml

```diff
@@ -33,3 +35,33 @@ jobs:
     - run: npm test
       env:
         CI: true
+
+  job_2:
+
+    name: Deploy to gh-pages when push to master branch
+
+    needs: job_1
```

Comment:
what does this mean?

Reply:
@huningxin Job2 is deploying to gh-pages when push to master branch, it depends on job1.